apps/vmq_bridge/priv/vmq_bridge.schema

```diff
@@ -59,6 +59,36 @@
                                                        hidden
                                                       ]}. 
 
+%% @doc Set the persistent queue option for this bridge. By default this is off, 
+%% meaning the queued messages are stored in memory only. If set to on, 
+%% queued messages will be stored persistently on disk. If you're using 
+%% persistency, make sure to configure the queue_dir option.
+{mapping, "vmq_bridge.tcp.$name.persistent_queue", "vmq_bridge.config", [
+                                                                 {default, off},
+                                                                 {datatype, flag},
+                                                                 {include_default, "br0"},
+                                                                 {commented, off}
+                                                                ]}. 
+{mapping, "vmq_bridge.ssl.$name.persistent_queue", "vmq_bridge.config", [
+                                                       {default, off},
+                                                       {datatype, flag},
+                                                       hidden
+                                                      ]}. 
+
+%% @doc Set the directory where queued messages for this bridge will be stored, if persistent_queue is set to on.
+%% It is recommended to set this to a unique directory. If you're using multiple persistent bridge instances, 
+%% then omitting this option can lead to serious problems. 
+{mapping, "vmq_bridge.tcp.$name.queue_dir", "vmq_bridge.config", [
+                                                             {datatype, string},
+                                                             {include_default, "br0"},
```

Comment:
directory?

Reply:
I think I should have been more precise, my suggestion was to use the datatype directory instead of string, to make it clear what is expected. queue_dir was just fine. Apologies for the confusion.

Reply:
Done.